src/qt/masternodelist.cpp

```diff
@@ -244,9 +249,14 @@ void MasternodeList::updateDIP3List()
 
         CTxDestination payeeDest;
         QString payeeStr = tr("UNKNOWN");
-        if (ExtractDestination(dmn.pdmnState->scriptPayout, payeeDest)) {
-            payeeStr = QString::fromStdString(EncodeDestination(payeeDest));
+        std::string payees = "UNKNOWN";
+        for (size_t i = 0; i < dmn.pdmnState->payoutShares.size(); i++) {
+            const auto& payeeShare = dmn.pdmnState->payoutShares[i];
+            if (ExtractDestination(payeeShare.scriptPayout, payeeDest)) {
+                payees = (i == 0) ? EncodeDestination(payeeDest) : (payees + ", " + EncodeDestination(payeeDest));
+            }
         }
+        payeeStr = QString::fromStdString(payees);
```

Comment:
if ExtractDestionation will fail the size of array would be wrong.
Let's add instead for each failed "UNKNOWN"

something like:
```
QString payeeStr;
for (const auto& payee : dmn.pdmnState->payoutShares) {
   if (!payeeStr.empty()) payeeStr += ", ";
   if (ExtractDestination(payeeShare.scriptPayout, payeeDest)) {
     payees += QString::fromStdString(EncodeDestination(payeeDest));
  else payees += tr("UNKNOWN");
}
```